pkg/scheduler/algorithm/predicates/utils.go

```diff
@@ -67,7 +67,7 @@ func FilterPodsByNamespace(pods []*v1.Pod, ns string) []*v1.Pod {
 
 // CreateSelectorFromLabels is used to define a selector that corresponds to the keys in a map.
 func CreateSelectorFromLabels(aL map[string]string) labels.Selector {
-	if aL == nil || len(aL) == 0 {
+	if len(aL) == 0 {
```

Comment:
`nil` can not be pasted to len()(Although len(nil)=0, explicit expression is better), I think the judgement of map `nil` or `empty` is necessary. 😝